src/components/Form/Form.vue

```diff
@@ -1,22 +1,22 @@
 <template>
   <form class="">
-    <select class="" name="">
-      <option>Choose a State</option>
+    <select class="" name="" @change="selectState">
+      <option selected disabled>Choose a State</option>
```

Comment:
We'll want to remember to come back to these an add in our class names for styling as well as any applicable aria tags where we currently have empty strings as placeholders.